src/psqlgraph/voided.py

```diff
@@ -0,0 +1,294 @@
+"""A module for recording various voided states within a graph."""
```

Comment:
Can I get a line of context here? Under what circumstances should/does a clinet void things in the graph?